lxd/instance/drivers/driver_lxc.go

```diff
@@ -8342,6 +8345,12 @@ func (d *lxc) Metrics(hostInterfaces []net.Interface) (*metrics.MetricSet, error
 		if err != nil && isRunning {
```

Comment:
When `!isRunning` causes an early return so maybe we should stop checking it afterward?

Reply:
fixed